lib/arjdbc/postgresql/base/array_encoder.rb

```diff
@@ -0,0 +1,21 @@
+# This implements a basic encoder to work around ActiveRecord's dependence on the pg gem
+module PG
+  module TextEncoder
+    class Array
+
+      def initialize(name:, delimiter:)
+        @type = if name == 'string[]'.freeze
```

Comment:
I called out string explicitly because it currently get's converted to 'character varying' which the driver didn't seem to be able to handle.